Automattic/jetpack
# VideoPress: Implement "Learn How" modal and notices for chapters

Closes #27229

#### Changes proposed in this Pull Request:

* VideoPress: add a notice and a modal informing the users how they can create chapters

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?


No.


#### Testing instructions:






* Add a VideoPress block and upload a video
* In the right sidebar you should see a new notice in the **details** section (as you can see in the screenshot below)

<img width="291" alt="image" src="https://user-images.githubusercontent.com/6406071/199618391-23c14a29-70a4-4cb7-8c4e-ed7ee08ac8c9.png">

* On this notice you can:
    * Click on the **X** icon to dismiss it and it will no longer appear (even after refreshing the page)
    * Click on "Learn how" to open a modal containing information to instruct users about they can create chapters
    
<img width="1724" alt="image" src="https://user-images.githubusercontent.com/6406071/199619065-05eca271-f667-47cc-8556-3a322679936b.png">
